oidc-server/server.go

```diff
@@ -17,6 +17,7 @@ func (s *OIDCServer) StartServer(debug bool) error {
 		return err
 	}
 
+	s.m.QueueUser(&s.config.MockUser)
```

Comment:
This is actually not necessary because the InfiniteMockUserMiddleware will add the default mock user anyway if the UserQueue is empty